src/workspaces/ShareWorkspaceModal/ShareWorkspaceModal.ts

```diff
@@ -97,7 +98,8 @@ const ShareWorkspaceModal: React.FC<ShareWorkspaceModalProps> = (props: ShareWor
   const addCollaborator = (collaboratorEmail) => {
     if (!validate.single(collaboratorEmail, { email: true, exclusion: aclEmails })) {
       setSearchValue('');
-      setAcl(append({ email: collaboratorEmail, accessLevel: 'READER' }));
+      // TODO: Why does the appended entry need to be cast?
+      setAcl(append({ email: collaboratorEmail, accessLevel: 'READER' } as AccessEntry));
```

Comment:
We are just going to show the default state for adding a user. I think `pending` should be marked as optional because only the server really knows that (we could mark a new user as "pending", but that will change the UI somewhat, and we already show new entries in green). And we should include `canCompute` and `canShare` with values of false since that is what we show as the state for a new user.

![image](https://github.com/DataBiosphere/terra-ui/assets/484484/f942ed39-e1ea-4881-a396-2fe6ed70da53)

Reply:
But we can also do this ourselves…. I'm approving from a Workspaces CODEOWNER.